framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2530,4 +2535,33 @@ private void printGitProperties() {
             System.out.println("IOException while reading git.properties: " + e.getMessage());
         }
     }
+
+    /**
+     * Extract the version of the Checker Framework
```

Comment:
If you use "Extract", say from where.  Even better, I would leave implementation details like that out of the specification and just use "Returns".